ts/features/itwallet/common/components/ItwUpcomingWalletBanner.tsx

```diff
@@ -1,32 +1,43 @@
 import React from "react";
-import { Banner } from "@pagopa/io-app-design-system";
-import { Linking } from "react-native";
+import { Banner, VSpacer } from "@pagopa/io-app-design-system";
 import I18n from "../../../../i18n";
 import { useIOSelector } from "../../../../store/hooks";
 import { trialStatusSelector } from "../../../trialSystem/store/reducers";
 import { itwTrialId } from "../../../../config";
 import { SubscriptionStateEnum } from "../../../../../definitions/trial_system/SubscriptionState";
 import { isItwEnabledSelector } from "../../../../store/reducers/backendStatus";
 
-export const ItwUpcomingWalletBanner = () => {
+type ItwUpcomingWalletBannerProps = {
+  vSpacer?: boolean;
```

Comment:
What about accepting something like: 
```
bottomSpacing?: IOSpacer
```
Does it make sense to you?

Reply:
Thanks for the suggestion